src/cli/discourse.js

```diff
@@ -97,7 +97,8 @@ const command: Command = async (args, std) => {
   await load(
     {
       project,
-      params: {weights},
+      params: null,
```

Comment:
Side-note: I feel like this is an example of why
`key?: $Shape<T>`
is better than the current
`key: ?$Shape<T>`
for an Options type.

But let's not change that in this PR.

Reply:
Ack.